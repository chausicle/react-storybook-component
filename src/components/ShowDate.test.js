import React from 'react'
import renderer from 'react-test-renderer'
import ShowDate from './ShowDate'

test('ShowDate shows correct date', () => {
  const tree = renderer.create(<ShowDate />).toJSON()
  expect(tree).toMatchSnapshot()
})
