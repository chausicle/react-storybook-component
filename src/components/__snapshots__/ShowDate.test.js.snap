// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowDate shows correct date 1`] = `
<div>
  <span>
    The correct date is: 2017-04-05
  </span>
</div>
`;
